src/test/resources/features/Tasks/Tasks.feature

```diff
@@ -486,3 +486,13 @@ Feature: Tasks
     Then 'Group And Sort' Tab is displayed
     When I select the 'Labels' filter type
     Then 'Labels' Tab is displayed
+    
+  @functional
+  Scenario:  Add Task in "TASKS" tab (task NOT belong to project)
+    Given I am authenticated as 'admin' random user
+    And I open the app center menu
+    And I open all application page
+    When I go to 'Tasks' application
+    And I select tasks tab
+    And I click on add new task button
+    Then The 'Add Task' drawer is displayed
```

Comment:
Can you complete the test by creating a task ? 
Just display the drawer is not a relevant test

Reply:
Done